src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.c

```diff
@@ -27,27 +30,220 @@ const ActorInit En_Horse_Zelda_InitVars = {
     (ActorFunc)EnHorseZelda_Update,
     (ActorFunc)EnHorseZelda_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/func_80A6D8D0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/func_80A6D918.s")
+static AnimationHeader* sAnimationHeaders[] = { 0x06007148 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/EnHorseZelda_Init.s")
+static f32 sAnimPlaybackSpeeds[] = { 0.66666666f };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/EnHorseZelda_Destroy.s")
+static ColliderCylinderInit_Set3 sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 40, 100, 0, { 0, 0, 0 } },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/func_80A6DC7C.s")
+static ColliderJntSphItemInit sJntSphItemsInit[1] = {
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+        { 13, { { 0, 0, 0 }, 20 }, 100 },
+    },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/func_80A6DCCC.s")
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x39, 0x12, COLSHAPE_JNTSPH },
+    1,
+    sJntSphItemsInit,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/func_80A6DD14.s")
+static CollisionCheckInfoInit sColChkInfoInit = { 0xA, 0x23, 0x64, 0xFE };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/func_80A6DDFC.s")
+typedef struct {
+    /* 0x0 */ Vec3s unk_0;
+    /* 0x6 */ u8 unk_6;
+} unknownStruct; // size = 0x7
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Horse_Zelda/func_80A6DE38.s")
+static unknownStruct D_80A6E240[] = {
+    { 63854, 65036, 12578, 0x07 }, { 62248, 65036, 13013, 0x07 }, { 60394, 65119, 11630, 0x07 },
+    { 59742, 65063, 9573, 0x07 },  { 58771, 65036, 8364, 0x07 },  { 58917, 65143, 6919, 0x07 },
+    { 60343, 124, 5433, 0x07 },    { 62566, 2, 4537, 0x07 },      { 62587, 65501, 4527, 0x07 },
```

Comment:
oh the conversion here did not take sign into account, most of these really big numbers should actually be negative. the tool i use to convert does this too lol, its annoying